cargo-geiger/src/format/table.rs

```diff
@@ -107,32 +107,117 @@ fn table_footer(
     output_format: OutputFormat,
     status: CrateDetectionStatus,
 ) -> ColoredString {
-    let fmt = |used: &Count, not_used: &Count| {
-        format!("{}/{}", used.unsafe_, used.unsafe_ + not_used.unsafe_)
-    };
-    let output = format!(
-        "{: <10} {: <12} {: <6} {: <7} {: <7}",
-        fmt(&used.functions, &not_used.functions),
-        fmt(&used.exprs, &not_used.exprs),
-        fmt(&used.item_impls, &not_used.item_impls),
-        fmt(&used.item_traits, &not_used.item_traits),
-        fmt(&used.methods, &not_used.methods),
-    );
-    colorize(&status, output_format, output)
+    match output_format {
+        OutputFormat::Ratio => {
```

Comment:
Would it be possible to split out two functions here, one for `OutputFormat::Ratio` and one for `_` with unit tests for each? It might be nice to have a function and tests for the ratio calculation too?